rotkehlchen/rotkehlchen.py

```diff
@@ -965,3 +967,26 @@ def create_oracle_cache(
             return  # only for cryptocompare for now
 
         self.cryptocompare.create_cache(from_asset, to_asset, purge_old)
+
+    def update_curve_pools_cache(self) -> None:
+        # Delete current cache. Need to do this in case curve removes some pools
```

Comment:
Add a function docstring here.

```suggestion
    def update_curve_pools_cache(self) -> None:
        """Fill me in. Specify what this function does **and** what exceptions it may raise."""
```